File: workspace/data-model/src/main/java/de/pfiva/data/model/PfivaConfigData.java
package de.pfiva.data.model;

import java.io.Serializable;

public class PfivaConfigData implements Serializable {

	private static final long serialVersionUID = 1L;
	
	private int id;
	private String key;
	private String value;
	
	public int getId() {
		return id;
	}
	public void setId(int id) {
		this.id = id;
	}
	public String getKey() {
		return key;
	}
	public void setKey(String key) {
		this.key = key;
	}
	public String getValue() {
		return value;
	}
	public void setValue(String value) {
		this.value = value;
	}
	@Override
	public String toString() {
		return "PfivaConfigData [id=" + id + ", key=" + key + ", value=" + value + "]";
	}
}

File: angular_workspace/pfiva-web/src/app/data-model/PfivaConfigData.ts
export class PfivaConfigData {
    private id: number;
    private key: string;
    private value: string;
    
    public get Id(): number {
		return this.id;
	}
	public set Id(value: number) {
		this.id = value;
    }
    public get Key(): string {
		return this.key;
	}
	public set Key(value: string) {
		this.key = value;
    }
    public get Value(): string {
		return this.value;
	}
	public set Value(value: string) {
		this.value = value;
	}
}